src/RegisterPage.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include <mtx/responses/register.hpp>
 #include <mtx/responses/well-known.hpp>
+#include <mtxclient/http/client.hpp>
```

Comment:
Do you actually need this include?

Reply:
That's for the definition of  mtx::http::Callback<mtx::responses::Register>`, which I think is needed to separate the callback for registration.

Reply:
Ah, right, I missed that, I guess that is fine.